modin/pandas/series.py

```diff
@@ -666,6 +665,19 @@ def f(x):
             return result.to_pandas().squeeze()
         return result
 
+    def transform(self, func, axis=0, *args, **kwargs):  # noqa: PR01, RT01, D200
+        """
+        Call ``func`` on self producing a `BasePandasDataset` with the same axis shape as self.
+        """
+        if isinstance(func, list):
+            # drop nonunique functions
```

Comment:
why do we want to do that?

Reply:
To align with pandas behavior instead of getting `*** pandas.errors.SpecificationError: Function names must be unique if there is no new column names assigned`:
```python
(Pdb) pandas.Series([0., 1., 4.]).transform(["sqrt", "sqrt"])    
   sqrt
0   0.0
1   1.0
2   2.0
```

Reply:
This feels like a worthy comment to be added in code...